src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -1348,6 +1361,65 @@ void primitive_inst::do_runtime_in_place_concat() {
     GPU_DEBUG_TRACE_DETAIL << "[In place concat] " << concat_inst->id() << ": can_be_optimized " << std::endl;
 }
 
+void primitive_inst::do_runtime_in_place_crop() {
+    OV_ITT_SCOPED_TASK(ov::intel_gpu::itt::domains::intel_gpu_plugin, openvino::itt::handle("do_runtime_in_place_crop: " + id()));
+    GPU_DEBUG_GET_INSTANCE(debug_config);
+    GPU_DEBUG_IF(debug_config->disable_runtime_buffer_fusing) {
+        return;
+    }
+
```

Comment:
Let's return when shape is not changed.

Reply:
Added check and return after update shape of user crop

Reply:
Update: because the padding is reset every iteration, the padding must be calculated and propagated even if the shape does not change.